src/amber/cli/commands/routes.cr

```diff
@@ -5,9 +5,10 @@ require "../helpers/sentry"
 module Amber::CLI
   class MainCommand < ::Cli::Supercommand
     class Routes < Command
-      RESOURCE_ROUTE_REGEX = /(\w+)\s+\"([^\"]+)\",\s*(\w+)(?:,\s*(\w+)\:\s*\[([^\]]+)\])?/
-      VERB_ROUTE_REGEX     = /(\w+)\s+\"([^\"]+)\",\s*(\w+),\s*:(\w+)/
-      PIPE_SCOPE_REGEX     = /routes\s+\:(\w+)(?:,\s+\"([^\"]+)\")?/
+      RESOURCE_ROUTE_REGEX = /(\w+)\s+\"([^\"]+)\",\s*([\w:]+)(?:,\s*(\w+)\:\s*\[([^\]]+)\])?/
+      VERB_ROUTE_REGEX     = /(\w+)\s+\"([^\"]+)\",\s*([\w:]+),\s*:(\w+)/
+      WEBSOCKET_ROUTE_REGEX= /(websocket)\s+\"([^\"]+)\",\s*([\w:]+)/
+      PIPE_SCOPE_REGEX     = /(routes)\s+\:(\w+)(?:,\s+\"([^\"]+)\")?/
```

Comment:
Actually, one last comment. Regexes this big are hard to reason about. Since Crystal supports PCRE extended, what about adding comments to make this more friendly to future devs?

Something like this: https://stackoverflow.com/questions/34578333/correct-way-to-add-comments-to-a-regular-expression-in-php

Reply:
I didn't write any of the Regexs. Many were already there and @docelic modified them slightly and added the websocket one.

It would take me a while to figure them out (to your point), then comment them.
@elorest and/or @docelic - would you be willing to add comments to the regular expressions here?

Reply:
Heya, my vote would be in favor of just committing as-is, because what we have is an improvement. Further incremental improvements can be done by other contributors if/when they want. But in case @elorest can spare some time for this, awesome!